.github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       github.event_name == 'workflow_dispatch' ||
       startsWith(github.event.comment.body, 'Run Python ValidatesContainer Dataflow ARM')
 
-    runs-on: [self-hosted, ubuntu-20.04, main]
```

Comment:
I think we should fix the self host image. I also remember the github secret does not work for now. cc: @damccorm Is Beam repo's secret fixed?

Reply:
This test was failing due to a known issue where the building_wheels step fails on self-hosted runners. This is a temporary solution; once we make progress on [this issue](https://github.com/apache/beam/issues/33506), we can revert the workflow back to self-hosted.

Reply:
> Is Beam repo's secret fixed?

Yeah, I fixed this.

In general, I'm fine moving to hosted runners for now until we figure out what is wrong with self-hosted.

Reply:
@Abacn let me know if you have other concerns, or feel free to merge